### ISSUE TITLE
klab fetch shouldn't focus

### ISSUE THREAD

**Issue by asymmetric** [dapphub/klab#389]
Or at least, it should make it possible not to focus.

The usecase is me trying to debug two proofs, and `fetch` stealing the focus away from the currently focus one seems unnecessary.

**Comment by asymmetric** [comment]
Yeah that's true. I just think it's a behaviour that isn't necessarily part of the semantics of a "fetch".

**Comment by asymmetric** [comment]
Alternatively, since it asks questions already, it could ask if it should focus and default to yes.

**Comment by xwvvvvwx** [comment]
> Alternatively, since it asks questions already, it could ask if it should focus and default to yes.

I would be OK with this :+1: :ok_hand:
